static/js/main.js

```diff
@@ -25,12 +25,8 @@
 
   langPickerTogglerElement.addEventListener('click', function () {
     langPickerElement.classList.toggle('hidden')
-
-    if (langPickerTogglerElement.getAttribute('aria-expanded') === 'true') {
-      langPickerTogglerElement.setAttribute('aria-expanded', 'false')
-    } else {
-      langPickerTogglerElement.setAttribute('aria-expanded', 'true')
-    }
+    const isAriaExpanded = langPickerTogglerElement.getAttribute('aria-expanded') === 'true'
+    langPickerTogglerElement.setAttribute('aria-expanded', (!isAriaExpanded).toString())
```

Comment:
I understand @Trott's point, the boolean/string conversions are not pretty. However, if I remember all those questionable DOM / WebIDL conversion specifications correctly, the whole `(...).toString()` is unnecessary:

```suggestion
    langPickerTogglerElement.setAttribute('aria-expanded', !isAriaExpanded)
```

(I luckily haven't had to write frontend JS in a while, so I might be wrong, but it seems to work in FF/IE/Chrome.)

Reply:
> I understand @Trott's point,

Clearly I'm in the minority on this so I'm fine with it landing.